tests/api_send/test_send_message.py

```diff
@@ -42,12 +44,37 @@ def test_send_message(self):
         self.assertTrue(is_T_HASH(result))
 
         # When having an optional property, nonce
+        sleep(2)
```

Comment:
It's option

I think that
before : sleep(2)

somewhere : 
sleep_time = 2
after : sleep(sleep_time)